### PR TITLE
Styling changes to README & typography fixes

### PR DESCRIPTION
**Pull request by Tobias-Hunter**
## Checklist
 - [ ]  A link to the issue/user story that the pull request relates to
 - [ ]  How to recreate the problem without the fix
 - [ ]  Design of the fix
 - [ ]  How to prove that the fix works
 - [ ]  Automated tests that prove the fix keeps on working
 - [ ]  Documentation - any JSDoc, website, or Stackoverflow answers?

## Steps to Reproduce

1.
2.
3.
4.

## Existing issues

- [ ] [Stack Overflow issues](http://stackoverflow.com/tags/hyperledger-composer)
- [ ] [GitHub Issues](https://github.com/hyperledger/composer/issues)
- [ ] [Rocket Chat history](https://chat.hyperledger.org/channel/composer)

### CODE REVIEW

File: packages/composer-playground/src/app/editor/editor.component.scss [hunk 1]
```diff
@@ -97,9 +97,30 @@ app-editor {
...
-      min-height: 63vh;
-      max-height: 63vh;
+      min-height: 72vh;
+      max-height: 72vh;
       overflow-y: auto;
+
+      pre {
+        font-size: 0.8rem;
+        color: $white;
+        background-color: #272B36;
```
Comment (nklincoln) [inline packages/composer-playground/src/app/editor/editor.component.scss]: global color?

File: packages/composer-playground/src/assets/styles/base/_codemirror.scss [hunk 2]
```diff
@@ -74,4 +74,8 @@
       flex:1;
       color: #8C9696;
```
Comment (nklincoln) [inline packages/composer-playground/src/assets/styles/base/_codemirror.scss]: global color

File: packages/composer-playground/src/app/editor/editor.component.scss (hunk 1 above)
Comment (Tobias-Hunter) [inline packages/composer-playground/src/app/editor/editor.component.scss]: This isn't a global colour - pulled from the documentation code blocks!

File: packages/composer-playground/src/assets/styles/base/_codemirror.scss (hunk 2 above)
Comment (Tobias-Hunter) [inline packages/composer-playground/src/assets/styles/base/_codemirror.scss]: Wasn't my change but I'll be good 😉

**Pull request merged**